### ISSUE TITLE
Data Preprocessing 

### ISSUE THREAD

**Issue by ZiAngGu1** [1zb/3DShape2VecSet#3]
Hi there,
    Thanks for the wonderful work, the performance is impressive!
    I just want to know could you guys provide the code for data preprocessing? 
    Thank you for your time,
Sincerely,
ZiAng

**Comment by zwb0** [comment]
Same needs here. Could author provide the data preprocessing codes to instruct me to generate occupancies.zip and surfaces.zip for my own dataset. Thanks a lot!

**Comment by Bathsheba** [comment]
Please give us a pointer here?
Or at least pull a sample datafile from each category? 
I cannot get the 550GB occupancies file because the server cannot maintain a connection long enough.
Thanks!

**Comment by StriveZs** [comment]
Same needs here. Due to my server can‘t afford this the 550G occupancies, so I want to use the data preprocessing script to convert my own data for training. @1zb  Could you please provide the data preprocessing script. Thx

**Comment by aakash-saboo** [comment]
Hello @Bathsheba @StriveZs @zwb0 @ZiAngGu1 
Did you guys get the preprocessing code? Same problem with me.

Thanks

**Comment by Bathsheba** [comment]
No, they never gave up anything, but eventually I was able to download some of the sample file and it turned out 7-zip can unpack a fragment of an archive. The file format is simple and it wasn't difficult to generate my own.

**Comment by wizardlf** [comment]
> No, they never gave up anything, but eventually I was able to download some of the sample file and it turned out 7-zip can unpack a fragment of an archive. The file format is simple and it wasn't difficult to generate my own.

can you show me the file format  i cant download and unzip it，thanks！

**Comment by chunyang-w** [comment]
> No, they never gave up anything, but eventually I was able to download some of the sample file and it turned out 7-zip can unpack a fragment of an archive. The file format is simple and it wasn't difficult to generate my own.

Hi Bathsheba, thanks for contributing to this issue. I am wondering do you mind share some sample files with us? I tried to download the file with their link, but is no longer available - (google drive issue probably). Thanks you for considering this; Have a nice day and happy coding!
